src/helpers/buildWalletSections.js

```diff
@@ -225,7 +225,6 @@ const coinEditContextMenu = (
   sortedAssets,
   balanceSectionData,
   isCoinListEdited,
-  currentAction,
```

Comment:
This is a bug, we don't pass through any `currentAction` parameter on usages of `coinEditContextMenu`

Reply:
@osdnk is this from your RLV changes?

Reply:
This is part of a path of unused code (the WalletScreen now uses the brief balance section selector instead, while the Showcase uses the old RLV but supplies a different data source so ends up not using this particular selector)

Reply:
I have a separate PR to remove this unused code but a few other things need to be taken care of

Reply:
Ah looks like this is the actual fix for the AddFundsInterstitial - bc this code path IS used for "isEmpty"

Reply:
lol, looks so messy, but this is it. Thanks